game-config/rock_paper_scissors.py

```diff
@@ -11,15 +11,15 @@ def rock_paper_scissors():
     var.win_count = 0
     i = var.NUMBER_OF_GAMES
     while i <= var.NUMBER_OF_GAMES:
-        if i == 0:
```

Comment:
This line was breaking the logic for when the computer wins 2 games after one loss the variable is equal to 0 which was breaking the for loop